NineChronicles.Headless/ActionEvaluationPublisher.cs

```diff
@@ -473,7 +473,7 @@ await using (var df = new DeflateStream(c, CompressionLevel.Fastest))
                                 }
 
                                 var compressed = c.ToArray();
-                                Log.Information(
+                                Log.Verbose(
```

Comment:
Is this log verbose too? (When I saw the logs you left, it seemed that only the below logs are verbose.)

```
this part

"[{ClientAddress}], #{BlockIndex}, {Action}," +
                                        " {EncodeElapsedMilliseconds}, {BroadcastElapsedMilliseconds}, {TotalElapsedMilliseconds}",
```

Reply:
Yes, this part appears pretty frequently too. I'll put it back to Info if it's necessary in the future :pray: